src/server_manager/web_app/app.ts

```diff
@@ -347,15 +347,17 @@ export class App {
     if (!digitalOceanAccount) {
       return [];
     }
+    let showedWarning = false;
     try {
       this.digitalOceanAccount = digitalOceanAccount;
       this.appRoot.digitalOceanAccount = {
         id: this.digitalOceanAccount.getId(),
         name: await this.digitalOceanAccount.getName(),
       };
       const status = await this.digitalOceanAccount.getStatus();
-      if (status !== digitalocean.Status.ACTIVE) {
-        return [];
+      if (status.warning) {
+        this.showDigitalOceanWarning(status);
+        showedWarning = true;
       }
       const servers = await this.digitalOceanAccount.listServers();
```

Comment:
I'm pretty sure this will throw if the account is in a bad state, that's why it was guarded.